integration_tests/tests/forseti/controls/scanner.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+control 'Scanner' do
+
+
+   describe "Run scanner" do
+
+       before :context do
+           command("forseti inventory purge 0").result
+           command("forseti inventory create").result
+           @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+           command("forseti model create --inventory_index_id #{@inventory_id} model_scanner").result
+           command("forseti model use model_scanner").result
+           command("sudo cp /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml /home/ubuntu/forseti-security/configs/forseti_conf_server_backup.yaml").result
```

Comment:
Why is a backup copy needed?  I don't think the audit logging scanner needs to be disabled.  Even if necessary, why not just re-reset the the yaml parameter instead of mv the file back?

Reply:
Backup copy is needed to restore the original state after executing the test. This makes tests independent of each other. If multiple changes are made in the config file, it wouldn't be feasible to revert changes one-by-one.